executor/src/test/java/org/k8loud/executor/service/MapperServiceImplTest.java

```diff
@@ -64,14 +64,16 @@ void testInvalidMapping(String collectionName, String actionName, Params params,
     }
 
     private static Stream<Arguments> provideMappableExecutionRQs() {
+        Params deleteResourceActionParams = new Params(Map.of("resourceName", "nameVal", "resourceType",
+                "typeVal", "namespace", "namespaceVal"));
         Params horizontalKubernetesScalingParams = new Params(
                 Map.of("resourceName", "nameVal", "resourceType", "typeVal", "namespace", "namespaceVal", "replicas",
                         "420"));
         Params horizontalOpenstackScalingParams = new Params(Map.of("region", "regionVal", "serverId", "EUNE"));
         Params verticalOpenstackScalingParams = new Params(
                 Map.of("region", "regionVal", "serverId", "EUNE", "flavorId", "asdad123312"));
 
-        return Stream.of(Arguments.of("kubernetes", "DeletePodAction", EMPTY_PARAMS),
+        return Stream.of(Arguments.of("kubernetes", "DeleteResourceAction", deleteResourceActionParams),
                 Arguments.of("kubernetes", "HorizontalScalingAction", horizontalKubernetesScalingParams),
                 Arguments.of("openstack", "HorizontalScalingAction", horizontalOpenstackScalingParams),
                 Arguments.of("openstack", "VerticalScalingUpAction", verticalOpenstackScalingParams)
```

Comment:
under this line, there is TODO with `UpdateConfigMapAction`. Can it be added to stream now?

Reply:
Since we're getting rid of using JSON representation I don't want to refactor Params yet.
I'd have to take the format into account, possibly change request files.
I'd rather work on it later, after @Agwen12 is done with changing architecture to monolith